docs/stack.md

```diff
@@ -53,7 +51,7 @@ Which simplified would be:
 
```

Comment:
**"Entity E can call Function F (Param P...) on App A only if Param P satisfies Rules R and Permission Manager PM can revoke or reassign that permission"**

In this statement is not clear:

What those Params are? What they represent?
Where are those Rules?
What is the Permission Manager? (it's also a little confusing name it PM)

Reply:
I've tried to make this a little more clear with 78dae21, although the "rules" are really hard to explain in that context (they get assigned when you create the permission).